app-web/src/utils/hooks.js

```diff
@@ -26,13 +26,22 @@ import isEqual from 'lodash/isEqual';
 export const useSearch = (query, staticIndex) => {
   const [Index, setIndex] = useState(null);
   const [results, setResults] = useState(null);
+  
   useEffect(
     () => {
       // load or create index
       if (Index === null) {
         setIndex(ElasticLunr.load(staticIndex));
       } else {
-        const searchResults = Index.search(query, { expand: true }).map(({ ref }) =>
+        const searchResults = Index.search(query, { fields: {
+          title: { boost: 4 },
+          content: { boost: 1 },
+          description: { boost: 1 },
+          collectionName: { boost: 2 },
+          labels: { boost: 2 },
```

Comment:
lets give labels a better boost since it is a field that authors dictate themselves